lib/HTML/FormBuilder.pm

```diff
@@ -71,32 +75,29 @@ sub new {
 
 #####################################################################
 # Usage      : Add a new fieldset to the form
-# Purpose    : Allow the form object to carry more than 1 fieldset
+# Purpose    : Allow the form object to carry more than 1 fieldsets
```

Comment:
Standard English grammar is the old way (fieldset agrees with "one" rather than "more than one" so it should be "more than one fieldset").